### PR TITLE
[DOCFIX] Update the metrics doc with cluster performance metrics

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: docs/en/operation/Metrics-System.md [hunk 1]
```diff
@@ -151,10 +151,19 @@ For more details about the metric types, please refer to [the metrics library do
 ## Master Web UI Metrics
 
 Besides the raw metrics shown via metrics servlet or custom metrics configuration,
-users can view more human-readable metrics stored in the leading master via leading master web UI metrics page.
+users can track key cluster performance metrics in a more human-readable way in the web interface of 
+Alluxio leading master (`http://<leading_master_host>:19999/metrics`).
 
+// TODO(lu) update graph
```
Comment (LuQQiu) [inline docs/en/operation/Metrics-System.md:157]: Taking a screenshot now

**Comment by LuQQiu** [comment]
@yuzhu @calvinjia PTAL the metrics doc changes, thanks!

File: docs/en/operation/Metrics-System.md [hunk 2]
```diff
@@ -175,6 +183,9 @@ Detailed descriptions of those metrics are in [cluster metrics]({{ '/en/referenc
 
 `Logical Operations` and `RPC Invocations` present parts of the [master metrics]({{ '/en/reference/Metrics-List.html' | relativize_url }}#master-metrics).
 
+`Saved Under FileSystem Operations` shows the operations fulfilled by Alluxio's namespace directly
+without accessing UFSes. Benefits could be significant if the target UFS is remote or slow in response.
```
Comment (calvinjia) [inline docs/en/operation/Metrics-System.md:187]: Could you also add that it can save costs if the underlying storage charges based on requests?

**Review by calvinjia — APPROVED** [review]

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**